scripts/release/package_audit.rb

```diff
@@ -347,10 +439,12 @@ def check_latest_container_version(client, containers, name)
 
 # Now check for latest versions
 client = HTTPClient.new
-check_latest_alpine(client)
-check_latest_container_version(client, containers, 'traefik')
-check_latest_minio(client, containers)
-check_latest_container_version(client, containers, 'redis')
+check_alpine(client)
+check_container_version(client, containers, 'traefik')
+check_minio(client, containers)
+check_container_version(client, containers, 'redis')
+base_pkgs = %w(regenerator-runtime single-spa vue vue-router vuetify vuex)
```

Comment:
These are the packages that the script upgrades to the versions listed in openc3-tool-base/package.json. The only remaining javascript is the various systemjs stuff which rarely changes so that can be manual. I also don't try to upgrade the roboto fonts (not sure how actually).